R/render.r

```diff
@@ -422,7 +422,22 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
-  deps <- bslib::bs_theme_dependencies(bs_theme)
+
+  # map secondary to component-active-bg
```

Comment:
Change suggested by @apreshill in https://github.com/r-lib/pkgdown/issues/1678#issuecomment-840760650